podpac/core/coordinates/stacked_coordinates.py

```diff
@@ -268,8 +290,8 @@ def name(self):
 
         Stacked dimension names are the individual `dims` joined by an underscore.
         """
-
-        return '_'.join(dim or '?' for dim in self.dims)
+        if any(self.dims):
+            return '_'.join(dim or '?' for dim in self.dims)
```

Comment:
This is a fine change, can you just change the test to match it?

Reply:
sorry, I should have caught that - i'm having trouble getting some tests to pass locally so I missed this one